CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+### [1.2.0] - 2021-10-05
+
+### Changed
+- Changed defualt values of parameters in all phases.  
```

Comment:
Explicitly write the changes in the log. The reasoning is that reader can understand the changes without reviewing the code